recipes/libheif/all/conanfile.py

```diff
@@ -26,6 +30,9 @@ def _source_subfolder(self):
     def config_options(self):
         if self.settings.os == 'Windows':
             del self.options.fPIC
+        if tools.Version(self.version) < "1.11.0":
+            del self.options.with_libaomav1
+            del self.options.with_dav1d
```

Comment:
```suggestion
        if tools.Version(self.version) < "1.10.0":
            del self.options.with_dav1d
```

libaom optional dependency available since in 1.7.0: https://github.com/strukturag/libheif/blob/82eccc96fc119351a0303357dd281707400cd376/CMakeLists.txt#L58
dav1d optional dependency introduced in 1.10.0: https://github.com/strukturag/libheif/blob/667eeabb553ce73094eb29faea3f31fb8610fec2/CMakeLists.txt#L68

Reply:
Enabled AOM support for version 1.9.1, thanks.

Support for dav1d appeared in 1.10.0 but an option `WITH_DAV1D` for easy control appeared only in 1.11.0. Since we do not intend to package intermediate versions, I prefer to enable them only for 1.11.0. Added a clarifying comment